### PR TITLE
coffeescript in js...

### PR DESCRIPTION
**Pull request by MathijsvVelde**
Are you going CoffeeScript on this?

### CODE REVIEW

**Comment by MaraniMatias** [comment]
I do not use CoffeeScript

File: readme.md [hunk 1]
```diff
@@ -128,12 +128,12 @@ module.exports = (grunt) => {
         }
       },
       custom:{
-        options: (platform,arch) => {
+        options: function (platform, arch) {
           return {
-            platform,
-            arch,
-            icon : './app/recursos/icon'// Auto detect platform extension
```
Comment (MathijsvVelde) [inline readme.md]: So, what is this then?

**Comment by MaraniMatias** [comment]
ES6 {
platform,
arch
}

ES5 {
platform:platform,
arch:arch
}

File: readme.md (hunk 1 above)
Comment (MaraniMatias) [inline readme.md]: ES6 {
platform,
arch
}

ES5 {
platform:platform,
arch:arch
}

**Pull request closed without merging**